src/main/charts/bitbucket/values.yaml

```diff
@@ -1553,3 +1553,59 @@ openshift:
   # configuration files as ConfigMaps.
   #
   runWithRestrictedSCC: false
+
+# OpenSearch config
+#
+opensearch:
```

Comment:
This might be far fetched but should we even call it `opensearch`? Should we call it `search` instead and the rest can be implementation detail?